internal/test/schemas/schemas.gen.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	Access_tokenScopes = "access_token.Scopes"
```

Comment:
I realise this is existing code, but would be nice to use a more Go-like name here :thinking: Maybe I'll raise a follow-up for this

Reply:
I think so, too.
Changing it would alter the existing variable name, leading to a breaking change, which is why I didn't address it in this PR.
I would appreciate it if you could raise a follow-up for this.

Maybe the following changes are needed:

```diff
diff --git a/pkg/codegen/codegen.go b/pkg/codegen/codegen.go
index 31978fe..1b0f478 100644
--- a/pkg/codegen/codegen.go
+++ b/pkg/codegen/codegen.go
@@ -639,7 +639,7 @@ func GenerateTypesForSecuritySchemes(t *template.Template, schemes map[string]*o
 
 	for _, schemeName := range SortedSecuritySchemeKeys(schemes) {
 		// Generate a type to be used as a key in context.WithValue
-		goTypeName := LowercaseFirstCharacter(SchemaNameToTypeName(schemeName)) + "ContextKey"
+		goTypeName := LowercaseFirstCharacter(ToCamelCase(SchemaNameToTypeName(schemeName))) + "ContextKey"
 		goType := Schema{
 			GoType:      "string",
 			Description: fmt.Sprintf("is the context key for %s security scheme", schemeName),
diff --git a/pkg/codegen/templates/constants.tmpl b/pkg/codegen/templates/constants.tmpl
index ca9fcb9..6704452 100644
--- a/pkg/codegen/templates/constants.tmpl
+++ b/pkg/codegen/templates/constants.tmpl
@@ -1,7 +1,7 @@
 {{- if gt (len .SecuritySchemeProviderNames) 0 }}
 const (
 {{range $ProviderName := .SecuritySchemeProviderNames}}
-    {{- $ProviderName | sanitizeGoIdentity | ucFirst}}Scopes {{$ProviderName | sanitizeGoIdentity | lcFirst}}ContextKey = "{{$ProviderName}}.Scopes"
+    {{- $ProviderName | sanitizeGoIdentity | camelCase | ucFirst}}Scopes {{$ProviderName | sanitizeGoIdentity | camelCase | lcFirst}}ContextKey = "{{$ProviderName}}.Scopes"
 {{end}}
 )
 {{end}}
```